### ISSUE TITLE
saber-link doesn't close when code is inside

### ISSUE THREAD

**Issue by Ir1d** [saberland/saber#277]
- [x] I confirm that this is an issue rather than a question.

## Bug report

```
[ `foo` ](/bar/)
```

I guess it has something to do with [here](https://github.com/saberland/saber/blob/master/packages/saber/lib/markdown/link-plugin.js#L48) , not so familiar yet so I can't open a PR. 

#### Version

```
$ npm list  | grep saber
└─┬ saber@0.6.9
```

#### What is expected?

```
<p><saber-link to="/bar/"> <code v-pre="">foo</code> </saber-link></p>
```

#### What is actually happening?

```
<p><saber-link to="/bar/"> <code v-pre="">foo</code> </a></p>
```

#### Other relevant information

- Result of running `saber -v`:
- Browser version (optional):
- Is Saber a global or local install?
- Which package manager did you use for the install?
- Does this issue occur when all plugins are disabled?

**Comment by lubien** [comment]
> I guess it has something to do with [here](https://github.com/saberland/saber/blob/master/packages/saber/lib/markdown/link-plugin.js#L48) [](/saberland/saber/blob/HEAD@{2019-06-22T02:16:26Z}/packages/saber/lib/markdown/link-plugin.js#L48) , not so familiar yet so I can't open a PR.

You're right. I think I got this.

**Comment by egoist** [comment]
Fixed in `saber@0.6.10`
